src/base58.c

```diff
@@ -26,50 +26,50 @@ static const uint32_t MAX_BUFFER_SIZE = 124;
 static const char BASE58ALPHABET[] = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
 
 size_t encode_base58(
-        const uint8_t *in, size_t length,
-        uint8_t *out, size_t maxoutlen
+        const uint8_t* inBuffer, size_t inSize,
+        uint8_t* outBuffer, size_t maxOutSize
 )
 {
-	uint8_t tmp[MAX_BUFFER_SIZE];
+	uint8_t tmpBuffer[MAX_BUFFER_SIZE];
 	uint8_t buffer[MAX_BUFFER_SIZE * 2];
 	size_t j;
 	size_t startAt;
 	size_t zeroCount = 0;
 
-	ASSERT(length <= MAX_BUFFER_SIZE);
+	ASSERT(inSize <= MAX_BUFFER_SIZE);
```

Comment:
I would prefer SIZEOF(tmpBuffer) here as we have two buffers with two different sizes in this function